komodo/insert_proposals.py

```diff
@@ -209,7 +209,7 @@ def create_pr_with_changes(
     pr_msg: str,
     rhel8: Optional[bool] = False,
 ):
-    commit_title = f"Add release {target}+rhel8" if rhel8 else f"Add release {target}"
+    commit_title = f"Add release {target} +rhel8" if rhel8 else f"Add release {target}"
```

Comment:
I think version numbers should have no spaces in them.

Can you consider using cut or other tools to extract the version number?
E.g.
```
COMMIT_MESSAGE="Add release 2024.02.04+rhel200"

release_name=$(echo $COMMIT_MESSAGE | rev | cut -d ' ' -f 1 | rev)  # remove: Add release
release_name=$(echo $release_name | cut -d + -f 1)  # remove: +rhel

echo $release_name
```